services/core-api/app/api/parties/party/models/sub_division_code.py

```diff
@@ -14,19 +14,6 @@ class SubDivisionCode(AuditMixin, Base):
     def __repr__(self):
         return '<SubDivisionCode %r>' % self.sub_division_code
 
-
     @classmethod
-    def all_options(cls):
```

Comment:
Why isn't this function needed anymore?

Reply:
This was formatting the dictionary for serialisation. it should just be querying objects and flask-restplus models handle the serialisation.